### ISSUE TITLE
Accessibility Testing for Medical Records on VA.gov (Allergies).

### ISSUE THREAD

**Issue by BobbyBaileyRB** [department-of-veterans-affairs/va.gov-team#65391]
### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

BobbyBaileyRB

### Use of color and color contrast checks

- [ ] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [X] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [X] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [X] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [X] Any charts, maps, infographics, and tables convey all information without only relying on color
- [X] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

Secondary buttons and Medications dropdown (print and download) button
[Medical records allergies](https://staging.va.gov/my-health/medical-records/allergies)
[VA Buttons](https://design.va.gov/components/button/)
The focus: state for the default secondary button has a foreground color of #0071BB and a background color of #DCE4EF. Running this through a contrast checker fails normal text at a ratio of 4.01:1.

### Who completed the axe scans?

BobbyBaileyRB

### axe checks

- [X] Each page has been scanned using axe (results shared below)
- [X] axe is integrated into your end-to-end testing

### axe DevTools scan results

[Allergies list page](https://staging.va.gov/my-health/medical-records/allergies)
Issues: Critical 5
IDs used in ARIA and labels must be unique

### Who completed the content zoom and reflow test?

BobbyBaileyRB

### Content zoom and reflow checks

- [X] All page elements are readable and usable at 200% zoom
- [X] All page elements are readable and usable at 300% zoom
- [X] All page elements are readable and usable at 400% zoom

### Who completed the keyboard navigation test?

BobbyBaileyRB

### Keyboard navigation checks

- [ ] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [ ] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [ ] All elements under focus have a visible focus indicator
- [X] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

**Improper HTML for link tag**
Section: Questions about this medical records tool, accordion "What if I have more questions" styled like a link but coded with a p-tag. Does not receive focus.
